configure

```diff
@@ -420,8 +426,8 @@ srcdir = $srcdir
 prefix = $prefix
 exec_prefix = $exec_prefix
 bindir = $bindir
-MANPREFIX = $prefix/share/man
-SHAREPREFIX = $prefix/share/vis
+mandir = $mandir
+sharedir = $sharedir
```

Comment:
But with this change `$MANPREFIX` and `$SHAREPREFIX` as used by the Makefile will once again be unset.